astro/src/content/docs/_shared/_licensed-edition-blurb-api.astro

```diff
@@ -0,0 +1,6 @@
+---
+const { feature } = Astro.props;
+---
+{feature && <><p><strong>Note:</strong> A license is required to utilize {feature}.</p></>}
+{!feature && <><p><strong>Note:</strong> A license is required to utilize <slot></slot>.</p></>}
```

Comment:
This seems overly complex. Can we get rid of either `feature` or `slot`?

Reply:
Removed slot. I don't see us ever using this component without passing in a feature.